content/en/apps/guides/security/securely-onboarding-users-at-scale.md

```diff
@@ -0,0 +1,101 @@
+
+## Spreadsheet use
+
+Most deployments manage users in a spreadsheet shared either in Google Docs or other cloud service. It is convenient to have a canonical shared location to access the data. This is an acceptable, but not ideal, solution as it ensures user account changes are instantly shared with all CHT admins, but can keep a number of key security requirements. For an ideal solution, [see "Ideal" below](#ideal-1-only-magic-links).
+
+Of paramount importance:
+
+* **Never** enable anonymous access to the URL for the spreadsheet. You **must always require authentication** to access user credential lists
+* Audit regularly who has access to the shared spreadsheet. Consider setting up an alert every time a new person is granted access.
+* Reduce the total number of users that need access by breaking up user lists by logical group. For example each sub-county could have its own user list. Only admins working for with sub-county would have access to the user list they work with.
+* Refrain from printing spreadsheets of users and passwords. They can be lost, stolen or easily photographed when shown in public.
+* If you never need to download a plaintext CSV with username and password, ensure the computer also has disk encryption enabled and requires a password to unlock.
+* Delete plaintext CSVs after you have used them for bulk upload. Do not keep plaintext copies on disk - redownload them from the authorized, authenticated cloud server as needed.
+
+## Transmitting credentials
+
+TK - only put in spreadsheet and share direct access. Do not send over Slack, Whatsapp, email or other medium. Use magic link instead.
+
+TK - when not using magic link, what is the realistic way to send credentials to a single user? 
+
+TK - when not using magic link, what is the realistic way to bulk provision 100s of devices without a printed list you can cross off? 
+
+TK - notes about downloading creds from [user management tool](https://github.com/medic/cht-user-management/) per [slack thread](https://medic.slack.com/archives/CHYAGKHN2/p1706744894943699?thread_ts=1706728984.849139&cid=CHYAGKHN2)? 
+
+## Example scenarios
+
+### Ideal practice 1: Only Magic Links
+
+Create a spreadsheet with full name, telephone number etc. Included is a username but NOT a password. When users are created in bulk via the command line or bulk user upload, have magic links sent to the user via an SMS gateway. This avoids the problem of passwords being stored in clear text in the spreadsheet or on a printed version.
+
+### Ideal practice 2: Unknown passwords, reset during provision
+
+An alternate and also secure approach, is to bulk create the users as described above, not use magic links, and use random passwords that you do not save. At a later time, manually change each user's password as you provision their mobile device. This also prevents a list of passwords from being stored in clear text.
```

Comment:
```suggestion
An alternate and also secure approach, is to bulk create the users as described above, not use magic links, and use random passwords that you do not save. Train the users on changing their password after they've logged in for the first time. This ensures if the initial password has leaked it's now no longer useful.
```

Reply:
ah interesting!   your edit involves the user, my original does not.  My thinking, and it may not scale to the field when provisioning 100s of devices, is to first bulk create all users then, as your provisioning each device, change the password one at a time. This is done entirely by the admins, and not known by the user.

Thoughts on which is more viable?

Reply:
When discussing with a large partner recently, we discovered that changing the password was already something that was included in training, although they just showed the screen and didn't actually submit the change. So it would fit nicely into their training to fill it in while they're there. For any partner who doesn't already include it it would be good to add because once they've done it if they ever need to change their password due to a breach they'll remember how to do it from last time.

Reply:
Gotcha - makes sense!  Let's go with your version.  thanks!